### ISSUE TITLE
[$25]List should open with keyboard and same list is currently opening from "Username" button and the image.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3246]
**Describe the bug**
List should open with keyboard and same list is currently opening from "Username" button and the image.

**To Reproduce**
1. Go   to the Challenges page: https://www.topcoder.com/challenges.      
2. Make sure the screen reader is switched on.      
3. Tab to Username and avatar

**Expected behavior**

Menu should open up once focused on either the username or the Avatar.

**Actual behavior**
"The menu does not open up by using keyboard: 
a. Dashboard 
b. My Profile 
c. Payment 
d. Setting 
e. Log Out

Also, this same list is currently opening from ""Username"" button and the image. It should get open from one of these only."

**Additional context**

Type: Navigation

Page: https://www.topcoder.com/challenges

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099922 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099922 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099922 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu making PR in sometime

**Comment by veshu** [comment]
@PrakashDurlabhji Ok please submit by next 3 hours, else we will open for pickup. Thanks!

**Comment by PrakashDurlabhji** [comment]
@veshu just few minutes now working actively

**Comment by PrakashDurlabhji** [comment]
@veshu it is done in https://github.com/topcoder-platform/community-app/pull/3261

**Comment by drasticdpk** [comment]
Hi @veshu and Team ,
1. Is it screen reader test ? if yes which screen reader should prefer ?
2. Expected behavior is quit confusion. We should treat "User name" and "Avatar" as different control? if yes currently while hovering the mouse of each control, menu list is opening.
3. Currently Search control is missing while reaching the tab on search icon. May be it is due to current changes.

**Comment by veshu** [comment]
@PrakashDurlabhji Please check this.

Based on above confirmation from @crazyk07, please see the summary I have made below:

> The username and profile pic should be treated as single element(i.e. when pressing tab we should highlight both and should show the menu options on tab(active state) and hover state.

> The search icon should be selected with tab and hover similar to combo of username and profile pic.

Let us know if it is not clear.

FYI @sexta13

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099922 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu just sometime else i will release myself confirm

**Comment by PrakashDurlabhji** [comment]
@veshu it is done https://github.com/topcoder-platform/community-app/pull/3275

**Comment by PrakashDurlabhji** [comment]
@veshu PR updated

**Comment by tosha5252** [comment]
Menu still does not open up when using the tab button

Win10/Chrome

**Comment by PrakashDurlabhji** [comment]
@veshu @tosha5252 verification screenshot it works fine now.
PR update done 
<img width="923" alt="3246_1" src="https://user-images.githubusercontent.com/28830240/64369121-4ae8e100-d039-11e9-8e30-f7d4938c7695.png">
<img width="880" alt="3246_2" src="https://user-images.githubusercontent.com/28830240/64369123-4ae8e100-d039-11e9-8a7c-b20d4fd79038.png">

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099922<br/><br/>```This is an automated message for crazyk via Topcoder X```
